vpr/src/util/vpr_utils.cpp

```diff
@@ -1767,7 +1818,7 @@ static void alloc_and_load_blk_pin_from_port_pin() {
     }
 
     /* Load the values */
-    /* itype starts from 1 since device_ctx.block_types[0] is the EMPTY_TYPE. */
+    /* itype starts from 1 since device_ctx.block_types[0] is the EMPTY_PHYSICAL_TILE_TYPE. */
     for (itype = 1; itype < device_ctx.logical_block_types.size(); itype++) {
```

Comment:
alloc_and_load_blk_pin_from_port_pin should be using `physical_tile_types` sizes, not `logical_block_types`.